### PR TITLE
align multi env doc with the new approach

### PR DESCRIPTION
**Pull request by tomermevorach**

### CODE REVIEW

**Comment by tomermevorach** [comment]
@gilhoffer done

File: docs/user_guide.md [hunk 1]
```diff
@@ -254,14 +254,15 @@ Lets stop and take a look at our workspace directory structure (for more info se
...
 	    — salesforce/      # specific config for Salesforce in the prod env
+        — static-resources/ # specific unique static resources for the dev env
 — salesforce/            # common cross-all-envs configuration for Salesforce
 — static-resources       # common static files for all environments
 
 ```
-Now, in a normal feature development flow we would do some changes to the dev env (e.g. by changing it directly in the service and running `fetch` (normal mode)), or by changing the **common** configuration and deploying to dev. After all tests in dev are done, we can go ahead and run:
+Now, in a normal feature development flow we would do some changes to the dev env (e.g. by changing it directly in the service and running `fetch` (normal mode)), or by changing the **common** configuration and deploying to dev. Do not forget to use the `salto move <elm-id-selectors> --to common` command in order to configure which elements should be common across all environemtns in the workspace (the `move` command can be executed at anytime, before or after changing the dev env).  After all tests in dev are done, we can go ahead and run:
```
Comment (gilhoffer) [inline docs/user_guide.md:265]: typo in "environemtns"
```suggestion
Now, in a normal feature development flow we would do some changes to the dev env (e.g. by changing it directly in the service and running `fetch` (normal mode)), or by changing the **common** configuration and deploying to dev. Do not forget to use the `salto move <elm-id-selectors> --to common` command in order to configure which elements should be common across all environemtns in the workspace (the `move` command can be executed at anytime, before or after changing the dev env).  After all tests in dev are done, we can go ahead and run:
```

File: docs/user_guide.md [hunk 2]
```diff
@@ -332,6 +333,21 @@ Manage your environments' services
 
 * `--env, -e` : The name of the environment to use
 
+### **salto element \<command> \<elm-id-selectors>**
+
+Manage your workspace environments
+
+**Arguments:**
+* `command` : The element management command [string] [required] [choices: "move", "copy"]
+* `elm-id-selectors` : A list of element id selectors [array]
```
Comment (gilhoffer) [inline docs/user_guide.md:342]: should we explain what are "element id selectors"?

**Comment by gilhoffer** [comment]
Should we add docs for the `diff` command?

File: docs/user_guide.md (hunk 2 above)
Comment (tomermevorach) [reply docs/user_guide.md:342]: Yes
How about adding a new 'salto element ids' subsection under 'advanced concepts'?

**Comment by tomermevorach** [comment]
Diff is still missing as well as copy, I'll add them both to the CLI manual

File: docs/user_guide.md (hunk 2 above)
Comment (gilhoffer) [reply docs/user_guide.md:342]: We explain element IDs at https://github.com/salto-io/salto/blob/master/docs/user_guide.md#references, might require some wording changes and proper link.

Comment (tomermevorach) [reply docs/user_guide.md:372]: Added a salto configuration elements' section under 'Advanced Concepts'

File: docs/user_guide.md [hunk 3]
```diff
@@ -254,14 +284,15 @@ Lets stop and take a look at our workspace directory structure (for more info se
...
 	    — salesforce/      # specific config for Salesforce in the prod env
+        — static-resources/ # specific unique static resources for the dev env
 — salesforce/            # common cross-all-envs configuration for Salesforce
 — static-resources       # common static files for all environments
 
 ```
-Now, in a normal feature development flow we would do some changes to the dev env (e.g. by changing it directly in the service and running `fetch` (normal mode)), or by changing the **common** configuration and deploying to dev. After all tests in dev are done, we can go ahead and run:
+Now, in a normal feature development flow we would do some changes to the dev env (e.g. by changing it directly in the service and running `fetch` (normal mode)), or by changing the **common** configuration and deploying to dev. Do not forget to use the `salto move <elm-id-selectors> --to common` command in order to configure which elements should be common across all environmetns in the workspace (the `move` command can be executed at anytime, before or after changing the dev env).  After all tests in dev are done, we can go ahead and run:
```
Comment (gilhoffer) [inline docs/user_guide.md:295]: The command here is missing `element` (salto *element* move)?

File: docs/user_guide.md [hunk 4]
```diff
@@ -254,14 +284,15 @@ Lets stop and take a look at our workspace directory structure (for more info se
 - envs/                  # folder for env specific configuration
     — dev/               # folder for the dev environment specific configuration
 	    — salesforce/      # specific config for Salesforce in the dev env
-	    — static-resources # specific unique static resources for the dev env
+	    — static-resources/ # specific unique static resources for the dev env
     — prod/              # folder for the prod environment specific configuration
 	    — salesforce/      # specific config for Salesforce in the prod env
+        — static-resources/ # specific unique static resources for the dev env
```
Comment (gilhoffer) [inline docs/user_guide.md:290]: indentation issue?

File: docs/user_guide.md [hunk 5]
```diff
@@ -332,6 +363,21 @@ Manage your environments' services
 
 * `--env, -e` : The name of the environment to use
 
+### **salto element \<command> \<elm-id-selectors>**
+
+Manage your workspace environments
+
+**Arguments:**
+* `command` : The element management command [string] [required] [choices: "move", "copy"]
```
Comment (gilhoffer) [inline docs/user_guide.md:371]: is it copy or clone eventually? :)

File: docs/user_guide.md (hunk 4 above)
Comment (tomermevorach) [reply docs/user_guide.md:290]: weird, in vscode it looks okay (also when rendering the md)

File: docs/user_guide.md (hunk 5 above)
Comment (tomermevorach) [reply docs/user_guide.md:371]: it is clone, fixing..

**Comment by tomermevorach** [comment]
@gilhoffer I have fixed all the errors you indicated in the last commit

**Review by gilhoffer — APPROVED** [review]

**Pull request merged**